### PR TITLE
add 2 people

### PR DESCRIPTION
**Pull request by reshamas**

### CODE REVIEW

**Comment by OriolAbril** [comment]
Can we use the blank person or even the sprint logo/banner from last time as placeholder image? Even for people for whom we know their pronouns, I don't think we should assume this link/relation between gender and appeareance. If we wanted to include that info we could add people pronouns if they feel comfortable doing so

**Comment by reshamas** [comment]
> Can we use the blank person or even the sprint logo/banner from last time as placeholder image? Even for people for whom we know their pronouns, I don't think we should assume this link/relation between gender and appeareance. If we wanted to include that info we could add people pronouns if they feel comfortable doing so

@OriolAbril 
yes, that's fine.  Let's use the gender-neutral placeholder image.

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 1]
```diff
@@ -99,4 +99,26 @@ São Paulo, São Paulo (Brazil)
 [{fab}`github`](https://github.com/vitaliset)
 [{fab}`linkedin-in`](https://www.linkedin.com/in/carlo-lemos)
 :::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_woman.jpg" class="rounded-circle">
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:104]:
```suggestion
<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
```

File: 2022-07_sprint/sprint_parties/contributors.md [hunk 2]
```diff
@@ -99,4 +99,26 @@ São Paulo, São Paulo (Brazil)
...
+Mariya
+
+(USA)
+
+[{fab}`github`](https://github.com/mariyayb)
+[{fab}`linkedin-in`](https://www.linkedin.com/in/mariya-b/)
+:::
+:::{grid-item}
+
+<img alt="Image for contributor template" src="../../_static/contributors/blank_man.jpg" class="rounded-circle">
```
Comment (reshamas) [inline 2022-07_sprint/sprint_parties/contributors.md:115]:
```suggestion
<img alt="Image for contributor template" src="../../_static/contributors/blank_person.jpg" class="rounded-circle">
```

**Comment by reshamas** [comment]
@OriolAbril @cluhmann 
Merging this in.

**Pull request merged**